examples/getting_started/flower/fedavg_flwr_cifar10.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvflare import FedJob
+from nvflare.app_opt.flower.controller import FlowerController
+from nvflare.app_opt.flower.executor import FlowerExecutor
+
+if __name__ == "__main__":
+    n_clients = 2
+    num_rounds = 2
+
+    job = FedJob(name="cifar10_flwr")
+
+    # Define the controller workflow and send to server
+    controller = FlowerController(server_app="server:app")
+    job.to_server(controller)
+
+    # Add flwr server code
+    job.to_server("server.py")
+
+    # Add clients
+    executor = FlowerExecutor(client_app="client:app")
+    job.to_clients(executor)
+
+    # Add flwr client code
+    job.to_clients("client.py")
```

Comment:
Adding @yhwen. Supporting placing of scripts to server/clients with `to()` is a general purpose solution. We have that built into the JobAPI to support any component that might refer to some code that needs to be deployed as part of the job. For example, now flower might also require a . toml file to be place at clients/server. 

So, to me it makes sense and is an extensible solution for different use cases.

Reply:
we could do it so python modules are supported directly
```
import client
job.to_clients(client)
```
but then you still cannot add config files that might be needed by a component.

Reply:
it's to_client**s**(). Only use when sending the component to all clients.

Reply:
The job.to(thing, target) syntax could be very confusing since the 1st arg could be many things. Perhaps we could improve by defining multiple input args, one arg for each type of things, like this:

job.to(scripts=[list of scripts], dirs=[list of dirs], objects=[list of objs], target=[list of targets])

It is possible or maybe even desirable to send multiple types of things in one call.